frontend/src/components/dashboard/create/govsg/GovsgRecipients.tsx

```diff
@@ -207,11 +205,7 @@ const GovsgRecipients = ({
           <FileInput isProcessing={isUploading} onFileSelected={uploadFile} />
           <p>or</p>
           <SampleCsv
-            params={
-              canAccessGovsgV
-                ? ['language', ...paramsWithoutPasscode]
-                : paramsWithoutPasscode
-            }
+            params={['language', ...paramsWithoutPasscode]}
```

Comment:
Does this mean that all sample CSVs will have a language field even if the template doesn't support multiple languages?

Reply:
Yes.

Reply:
I added a commit which selectively shows language column. Note there's no backend check.